utils/p4runtime_lib/simple_controller.py

```diff
@@ -146,6 +149,26 @@ def program_switch(addr, device_id, sw_conf_file, workdir, proto_dump_fpath):
         sw.shutdown()
 
 
+def validateTableEntry(flow, p4info_helper, runtime_json):
+    table_name = flow['table']
+    match_fields = flow.get('match')  # None if not found
+    priority = flow.get('priority')  # None if not found
+    match_types_with_priority = [
+        p4info_pb2.MatchField.TERNARY,
+        p4info_pb2.MatchField.RANGE
+    ]
```

Comment:
also optional match_kind, yes?

Reply:
I will go ahead and approve this, since this is easy to fix in a later separate PR, one that doesn't have two other open PRs based upon it already :-)